molgenis-semantic-mapper/src/main/resources/l10n/semantic-mapper_en.properties

```diff
@@ -0,0 +1,8 @@
+M01=No mapping project found with ID ''{0}''
+M02=Can't add mapping result to package ''{0}'' because it is a system package
+M03=''{0}'' can't be converted to type ''{1}''
+M04=Target repository does not contain the following attribute: ''{0}''
+M05=Attribute ''{0,label}'' in the mapping target is type ''{1}'' while attribute ''{2,label}'' in the target repository is type ''{3}''. Please make sure the types are the same
+M06=In the mapping target, attribute ''{0,label}'' of type ''{1}'' has reference entity ''{2,label}'' while in the target repository attribute ''{3,label}'' of type ''{4}'' has reference entity ''{5,label}''. Please make sure the reference entities of your mapping target are pointing towards the same reference entities as your target repository
+M07=Mapping project ''{0}'' already exists
+M08=Applying an algorithm on a null source value caused an exception. Is the target attribute required?
```

Comment:
unused according to intellij

Reply:
assign

Reply:
@fdlk @tommydeboer I can't find this property. One of you happened to fix this already and forgot to assign yourself to the review comment?